lib/ts/recipe/session/utils.ts

```diff
@@ -326,6 +324,13 @@ export async function validateClaimsInPayload(
     return validationErrors;
 }
 
-function defaultGetTokenTransferMethod({ req }: { req: BaseRequest }): "cookie" | "header" {
-    return getAuthModeFromHeader(req) === "header" ? "header" : "cookie";
+function defaultGetTokenTransferMethod({ req }: { req: BaseRequest }): "cookie" | "header" | "MISSING_AUTH_HEADER" {
+    switch (getAuthModeFromHeader(req)) {
```

Comment:
why is getAuthModeFromHeader not defined in the session recipe? Is it used in ts/supertokens.ts file?